components/kyma-environment-broker/internal/provider/owncluster_provider.go

```diff
@@ -0,0 +1,25 @@
+package provider
+
+import (
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal"
+	"github.com/kyma-project/control-plane/components/provisioner/pkg/gqlschema"
+)
+
+type NoHyperscalerInput struct {
+}
+
+func (p *NoHyperscalerInput) Defaults() *gqlschema.ClusterConfigInput {
+	return &gqlschema.ClusterConfigInput{}
+}
+
+func (p *NoHyperscalerInput) ApplyParameters(input *gqlschema.ClusterConfigInput, pp internal.ProvisioningParameters) {
+	// TODO: validate and merge kubeconfig here?
```

Comment:
this method must be empty because, please remove that comment. For other plans we are updating autoscaler params or regions if necessary, but it does not fit for own cluster plan.

Reply:
Removed